ui/src/app/styles/colors.ts

```diff
@@ -33,7 +33,13 @@ export default {
     'READER': '#8F8E8F',
     'WRITER': '#92B572'
   },
-  chartColors: ['#216FB4', '#6CACE4', '#8BC990', '#F8C954', '#F7981C', '#F0718B', '#F38D7A', '#A27DB7', '#CAB2D6']
+  chartColors: ['#216FB4', '#6CACE4', '#8BC990', '#F8C954', '#F7981C', '#F0718B', '#F38D7A', '#A27DB7', '#CAB2D6'],
+  // High-contrast colors for runtime status icon
+  runtimeStarting: '#9FFF00',
```

Comment:
nit: maybe nest these as was done from some of the otheruse-case specific colors:

```
runtimeStatus: {
  running: ...
}
```